homeassistant/components/utility_meter/sensor.py

```diff
@@ -140,6 +140,8 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class UtilityMeterSensor(RestoreEntity, SensorEntity):
     """Representation of an utility meter sensor."""
 
+    _no_check_last_reset = True
```

Comment:
I'd rather just write directly to extra state attributes. This doesn't feel like a proper interface that we want to support.